master/internal/rm/kubernetesrm/kubernetes_resource_manager.go

```diff
@@ -110,7 +107,7 @@ func New(
 	if err != nil {
 		return nil, err
 	}
-
+	log.Print(k.config.DefaultNamespace)
```

Comment:
nit: I think this was left here by mistake? If we want to keep it, we should probably format it for the user?

Reply:
yeah, sorry I forgot to remove debugging statements